### PR TITLE
fix todo: Move isDecremented to pkg/apis/core/validation

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
fix todo: Move isDecremented in "k8s.io/kubernetes/pkg/apis/core/validation"
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

2. If no release note is required, just write "NONE".
-->
```release-note
NONE
```

### CODE REVIEW

**Comment by hanxiaoshuai** [comment]
cc @hzxuzhonghu

File: pkg/apis/core/validation/validation.go [hunk 1]
```diff
@@ -4939,3 +4939,13 @@ func ValidateCIDR(cidr string) (*net.IPNet, error) {
 	}
 	return net, nil
 }
+
+func IsDecremented(update, old *int32) bool {
```
Comment (thockin) [inline pkg/apis/core/validation/validation.go]: PLEASE add comments for public functions

Comment (liggitt) [reply pkg/apis/core/validation/validation.go]: agree. thought this was a straight move and didn't notice it was not exported previously.

Comment (hanxiaoshuai) [reply pkg/apis/core/validation/validation.go]: OK,I have add comments  in new pr

**Comment by hanxiaoshuai** [comment]
@liggitt @thockin OK ,I have add comments in new [#59541](https://github.com/kubernetes/kubernetes/pull/59541)

**Pull request merged**